app/grandchallenge/components/models.py

```diff
@@ -1929,3 +1929,87 @@ def update_size_in_storage(self):
         else:
             self.size_in_storage = self.image.size
             self.size_in_registry = self.calculate_size_in_registry()
+
+
+class CIVForObjectMixin:
```

Comment:
Could you make these methods keyword arg only (`self, *, ...`)? Their signatures are quite complex.